SPL/SPL_c1.php

```diff
@@ -1553,7 +1553,7 @@ public function compare ($priority1, $priority2) {}
          * @param mixed $priority <p>
          * The associated priority.
          * </p>
-         * @return void 
+         * @return bool 
```

Comment:
```suggestion
         * @return true
```
@andrey-sokolov I suppose you have write access to accept sugesstion too